test/math.cc

```diff
@@ -88,7 +88,7 @@ double uniform_random(double a, double b)
 #define TEST_MATH_FUNCTION(F, f) TEST_MATH_FUNCTION_RANGE(F, f, -FLT_MAX, FLT_MAX)
 #define TEST_MATH_FUNCTION_ONE(F, f) TEST_MATH_FUNCTION_RANGE(F, f, -1.0, 1.0)
 #define TEST_MATH_FUNCTION_NEG(F, f) TEST_MATH_FUNCTION_RANGE(F, f, -1.0, 0.0)
-#define TEST_MATH_FUNCTION_POS(F, f) TEST_MATH_FUNCTION_RANGE(F, f, 0.0, 1.0)
+#define TEST_MATH_FUNCTION_POS(F, f) TEST_MATH_FUNCTION_RANGE(F, f, FLT_MIN, FLT_MAX)
```

Comment:
I think this will cause problems with functions like `Vc::log()`. I've seen discrepancies with the standard functions in `Vc::sin()` and `Vc::cos()` for large input values. We may need to revert this. I'm fine with leaving the change in, though, because we can replace with the standard equivalents if Vc functions fail.

Reply:
All right. Noted...